packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -107,6 +107,14 @@ export namespace ZoweExplorerApi {
          * @returns {Promise<boolean>}
          */
         isFileTagBinOrAscii(ussFilePath: string): Promise<boolean>;
+        /**
+         * Utility operations for USS files or directories.
+         *
+         * @param {string} outputPath the output/destination path for the file/directory
+         * @param {object} options Options for the file utility API endpoint
+         * @returns {Promise<Buffer>}
+         */
+        fileUtils(outputPath: string, options: object): Promise<Buffer>;
```

Comment:
Should this be optional?

Reply:
Thanks for pointing that out!

For `fileUtils`, we can make the object optional - however, I noticed that the function I am calling inside `fileUtils` (`zowe.Utilities.putUSSPayload`) expects the object to not be `null, undefined, or {}`:

```
ImperativeExpect.toNotBeNullOrUndefined(payload, ZosFilesMessages.missingPayload.message);
ImperativeExpect.toNotBeEqual(payload, {}, ZosFilesMessages.missingPayload.message);
```

Would it be best to update `zowe.Utilities.putUSSPayload` in the CLI so that `payload` is using an interface instead of being typed as `any`? This way, once the interface is implemented, we can remove the above checks since the interface would never be `null`, `undefined`, or `{}`. And then we can update the function prototype here to use that new interface.